x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/expression/function/SqlFunctionRegistry.java

```diff
@@ -177,6 +178,7 @@ public SqlFunctionRegistry() {
                 def(MonthName.class, MonthName::new, "MONTH_NAME", "MONTHNAME"),
                 def(MonthOfYear.class, MonthOfYear::new, "MONTH_OF_YEAR", "MONTH"),
                 def(SecondOfMinute.class, SecondOfMinute::new, "SECOND_OF_MINUTE", "SECOND"),
+                def(DateTimeFormat.class, DateTimeFormat::new, "DATETIME_FORMAT"),
```

Comment:
Any particular reason for not using the alphabetical order (as it is now in the registry)?

Reply:
Good catch, will fix, It was because of the original name `ToChar`.